README.md

```diff
@@ -72,6 +72,8 @@ yarn run bootstrap
 yarn run start
 ```
 
+> If your development environment is macOS, ensure that you have iTerm or iTerm2 installed. Otherwise, the command fails.
```

Comment:
see: https://github.com/netless-io/flat/blob/af8cc68d18189b0533a58558a52db91709f17945/scripts/launch/start.scpt#L12-L17

They do not report errors